src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionResolveHandler.cs

```diff
@@ -62,12 +63,17 @@ public TextDocumentIdentifier GetTextDocumentIdentifier(LSP.CodeAction request)
 
         public async Task<LSP.CodeAction> HandleRequestAsync(LSP.CodeAction codeAction, RequestContext context, CancellationToken cancellationToken)
         {
-            var document = context.GetRequiredDocument();
-            var solution = document.Project.Solution;
-
             var data = ((JToken)codeAction.Data!).ToObject<CodeActionResolveData>();
             Assumes.Present(data);
 
+            if (data.FixAllFlavors is not null)
```

Comment:
should add a comment here explaining that the fix all code action doesn't need further resolution (it already has the command callback when we create the original action)